### PR TITLE
Show database name when editing permission

### PR DESCRIPTION
**Pull request by WatanabeToshimitsu**
## What?
Show database name when editing permission

## Why?
To imporove UX

## See also
Resolves https://github.com/dataware-tools/dataware-tools/issues/103

### CODE REVIEW

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/app-user-manager/pull/236#" title="Author self-approved">WatanabeToshimitsu</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fapp-user-manager).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/app-user-manager/blob/master/OWNERS)~~ [WatanabeToshimitsu]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/components/organisms/PermissionListItem.tsx
Comment (yusukefs) [inline src/components/organisms/PermissionListItem.tsx:96]: このあたり消し忘れでしょうか？その他はLGTMです！

**Review by yusukefs — COMMENTED** [review]
↓のrenderOptionを使えば
https://mui.com/components/autocomplete/#country-select

database_id
**データベース表示名** （強調）

みたいな感じで見やすくなるかと思いました。（複数行になったときに問題が出るかもしれませんが・・・）
このあたりはまた別PRでも良さそうですね。

File: src/components/organisms/PermissionListItem.tsx
Comment (WatanabeToshimitsu) [reply src/components/organisms/PermissionListItem.tsx]: 修正しました！あざます！

**Comment by WatanabeToshimitsu** [comment]
ここの表示に関しては，あまりに名前を強調しすぎると機能との齟齬がある気がしていて，その辺り皆さんのご意見いただきたいですね....
とりあえず別PRで対応したいと思ってます！

**Comment by yusukefs** [comment]
たしかにIDのほうがメインですしIDを強調するとかも考えられますね。
また打ち合わせのときにお話しましょう！

**Review by yusukefs — APPROVED** [review]
LGTM!

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/app-user-manager/pull/236#" title="Author self-approved">WatanabeToshimitsu</a>*, *<a href="https://github.com/dataware-tools/app-user-manager/pull/236#pullrequestreview-849904949" title="Approved">yusukefs</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fapp-user-manager).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/app-user-manager/blob/master/OWNERS)~~ [WatanabeToshimitsu,yusukefs]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
